### PR TITLE
feat: bump pkg docker/library/kong@3.8

### PR DESCRIPTION
**Pull request by Venkat-Bramhasani**
**What problem does this PR solve?**:

feat:bump pkg:docker/library/kong@3.8

**Which issue(s) does this PR fix?**:

https://jira.nutanix.com/browse/NCN-104096

### CODE REVIEW

**Comment by mhrabovcin** [comment]
@Venkat-Bramhasani could we try to bump the whole kubernetes dashboard app to 7.10 - https://artifacthub.io/packages/helm/k8s-dashboard/kubernetes-dashboard

**Comment by Venkat-Bramhasani** [comment]
Hi @mhrabovcin , When I try to change the Kong version to 3.8, it automatically redirects to version 3.6 instead. After that, we're encountering the following error. Could you please suggest if I might have missed something?

2024-11-14 18:23:57 ERR failed to locate mapping for the following images:    err=<nil>
2024-[11](https://github.com/mesosphere/kommander-applications/actions/runs/11843191414/job/33003565804?pr=2823#step:7:12)-14 18:23:57 ERR docker.io/library/kong:3.6    err=<nil>
2024-11-14 18:23:57 INF   ^ hint: image mapping for another version exists in "licenses.d2iq.yaml":
2024-11-14 18:23:57 INF       container_image: docker.io/library/kong:3.8.0
      sources:
        - url: https://github.com/Kong/kong
          ref: 3.8.0
          license_path: LICENSE

**Comment by Venkat-Bramhasani** [comment]
Hi @mhrabovcin , Could you please help me figure out if I missed something while updating the Kong version? After upgrading, it's automatically selecting version 3.6 instead of 3.8.0. Any suggestions would be greatly appreciated!

**Comment by Venkat-Bramhasani** [comment]
Hi @mhrabovcin , can you please have a look into this PR and take the further action asap.

**Comment by mhrabovcin** [comment]
@Venkat-Bramhasani thanks, could  you now open kommander PR to get this change tested?

File: charts/kommander-operator/values.yaml [hunk 1]
```diff
@@ -12,6 +12,7 @@ kommanderoperator:
 kubetools:
   image:
     repository: mesosphere/kommander2-kubetools
-    tag:
+    tag: ""
+    pullPolicy: IfNotPresent
```
[the remark this answers is not in the document]
Comment (Venkat-Bramhasani) [reply charts/kommander-operator/values.yaml]: Suggested modification done.

**Review by mhrabovcin — APPROVED** [review]

**Pull request merged**